x-pack/plugins/synthetics/public/apps/synthetics/state/monitor_details/index.ts

```diff
@@ -95,6 +96,11 @@ export const monitorDetailsReducer = createReducer(initialState, (builder) => {
     .addCase(getMonitorAction.fail, (state, action) => {
       state.error = action.payload;
       state.syntheticsMonitorLoading = false;
+    })
+    .addCase(enableMonitorAlertAction.success, (state, action) => {
+      if ('updated_at' in action.payload && state.syntheticsMonitor) {
```

Comment:
Why are we relying on `updated_at`? Is this supposed to be a proxy for success? What if it's undefined? Seems like the success case shouldn't even run if there was an error preventing this field from being applied to the monitor?

Reply:
in case of errors, there is a separate response, so it's pretty safe to check for inferring type using `updated_at`